pyproject.toml

```diff
@@ -185,6 +185,7 @@ ignore = [
   "EM103",  # Exception must not use a `.format()` string directly, assign to variable first
   "C408",   # Unnecessary `dict` call (rewrite as a literal)
   "SIM117", # Use a single `with` statement with multiple contexts instead of nested `with` statements
+  "B005",   # Using `.strip()` with multi-character strings is misleading
```

Comment:
How do you feel if we kept this error as one that gets raised and made the change in the test provided above?

```suggestion
```